chapters/packages.tex

```diff
@@ -63,22 +63,22 @@ \subsection{Importing Definitions from a Package}\doublelabel{importing-definiti
 %TODO-FORMAT Should be formatted using tabs or tabular?
 An \lstinline!import!-clause can occur in one of the following syntactic forms:
 
-% Note a syntactic form cannot be written as a single \lstinline[mathescape=true] due to LaTeXML problem reported here:
+% Note a syntactic form cannot be written as a single \lstinline due to LaTeXML problem reported here:
 %   https://github.com/brucemiller/LaTeXML/issues/1274 (marked as fixed as of commit 80d7940)
 % Instead, we need to break up the \lstinline at each piece of math (taking the opportunity to leave the whitespace
 % outside both \lstinline and math).
 
-\lstinline[mathescape=true]!import $\mathit{definitionname}$;! (qualified import of top-level definition)
+\lstinline!import $\mathit{definitionname}$;! (qualified import of top-level definition)
 
-\lstinline[mathescape=true]!import $\mathit{packagename}$.$\mathit{definitionname}$;! (qualified import)
+\lstinline!import $\mathit{packagename}$.$\mathit{definitionname}$;! (qualified import)
 
-\lstinline[mathescape=true]!import $\mathit{packagename}$.{$\mathit{def}_{1}$, $\mathit{def}_{2}$, $\ldots$, $\mathit{def}_{n}$};! (multiple definition import)
+\lstinline!import $\mathit{packagename}$.{$\mathit{def}_{1}$, $\mathit{def}_{2}$, $\ldots$, $\mathit{def}_{n}$};! (multiple definition import)
 
-\lstinline[mathescape=true]!import $\mathit{packagename}$.*;! (unqualified import)
+\lstinline!import $\mathit{packagename}$.*;! (unqualified import)
 
-\lstinline[mathescape=true]!import $\mathit{shortname}$ = $\mathit{definitionname}$;! (renaming import of top-level definition)
+\lstinline!import $\mathit{shortname}$ = $\mathit{definitionname}$;! (renaming import of top-level definition)
 
-\lstinline[mathescape=true]!import $\mathit{shortname}$ = $\mathit{packagename}$.$\mathit{definitionname}$;! (renaming import)
+\lstinline!import $\mathit{shortname}$ = $\mathit{packagename}$.$\mathit{definitionname}$;! (renaming import)
```

Comment:
Currently I can only get it to work if I write:
`\lstinline!import $\mathit{shortname}$ = !\lstinline!$\mathit{packagename}$.!\lstinline!$\mathit{definitionname}$;! (renaming import)`

Reply:
Fixed (using one less break, but more aggressively commented).  Checked based on locally generated HTML.  Resolving.